pkg/scheduler/core/scheduling_queue.go

```diff
@@ -43,6 +43,10 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/util"
 )
 
+const (
+	queueClosed = "scheduling queue is closed"
```

Comment:
I don't think this really warrants a const.

Reply:
maybe an error var is better, similar to `FIFOClosedError`.

Reply:
In theory, yes, but we don't currently check the error anywhere. YAGNI.

Reply:
let's make it a var.